geomet_data_registry/tileindex/elasticsearch_.py

```diff
@@ -240,6 +240,58 @@ def add(self, identifier, data):
 
         return status_code
 
+    def bulk_add(self, data):
+        """
+        Add many items to the tileindex
+
+        :param data: GeoJSON dict
+
+        :returns: list of dict {layer_id: HTTP status code}
+        """
+
+        status_code = None
+
+        LOGGER.debug('Starting bulk add')
+        try:
+            # FIXME: refresh='wait_for' essentially blocks further processing
+            # until the document is fully available in ES.  This is done in
+            # order to make updates to the layer registration datetime
+            # immediately after in support of tracking performance (kind of
+            # ironic eh?).
+            # TODO: update using asyncio or multiprocessing
+
+            # add 'pipeline': 'register_datetime' to data_es
+            bulk_data = []
+            for doc in data:
+                op_dict = {
+                    'index': {
+                        '_index': self.name,
+                        '_type': '_doc'
+                    }
+                }
+                op_dict['index']['_id'] = doc['properties']['identifier']
+                bulk_data.append(op_dict)
+                bulk_data.append(doc)
+            r = self.es.bulk(index=self.name, body=bulk_data,
+                             pipeline='gdr_register_datetime')
```

Comment:
call this `gdr_register_datetime_pipeline`

Reply:
If we change it here we will have to change the pipeline name in `ElasticsearchTileIndex.setup()`, `ElasticsearchTileIndex.teardown()` and `ElasticsearchTileIndex.add()`.

Reply:
I can do the change and update the names in those files as well. 

@tomkralidis should I go on and update those other files ?